### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by picatchoof** [piratfm/eti-tools#43]
Dear All Hello,

I have tested the _mpe2ts_ and the _dvb-ip-mpe2ts_ for the ts file on the linke below and I received the Error :
**Segmentation fault (core dumped)**

the Link to download the TS
https://we.tl/t-oXk3uKhFvc

Some tests are done previously on some samples are Okey

Thanx for Help.

**Comment by andimik** [comment]
What transponder and which satellite is that? Your file is just mentioned sample.ts

I could test it with my dish.

**Comment by andimik** [comment]
Next time (also in other issues!) please always provide more information.

Oh, it seems this is from West African beam on Eutelsat 7 West  (= too far away from Central Europe) which you were mentioning in a TS Duck issue.

I found these streams:

![grafik](https://user-images.githubusercontent.com/24510556/127483335-cbf8c698-c198-45cf-8b1c-9bd9b2c58eea.png)

```
cat /tmp/7west.ts | dvb-ip-mpe2ts -p 3002 -i - -o - -a 224.30.2.117 -n 6000

```
will indeed crash. But

```
$ cat /tmp/7west.ts | mpe2ts 3002 224.30.2.117 6000 | vlc -
```

will bring some fragments (it does NOT crash) and vlc says

![grafik](https://user-images.githubusercontent.com/24510556/127485112-824c7515-5f67-44fa-ba03-d6df08917e35.png)


Then I've analyzed the sample.ts and found the following texts:

```
#EXTM3U
#EXT-X-VERSION:5
## Created with Broadpeak BkS350 Origin Packager(version=1.9.5)
#EXT-X-MEDIA-SEQUENCE:271236998
XT-X-INDEPENDENT-SEGMENTS
#EXT-X-TARGETDURATION:6
#USP-X-TIMESTAMP-MAP:MPEGTS=1005071036,LOCAL=2021-07-27T21:39:47.769200Z
#EXT-X-PROGRAM-DATE-TIME:2021-07-27T21:39:47.769205Z
#EXTINF:G

```
which then points me to 

https://www.nexttv.com/post-type-the-wire/broadpeak-performs-world-first-video-unified-packaging-and-encryption-of-dash-and-hls

Will this help you?

**Comment by andimik** [comment]
Well, now I tried under Windows and recognized that this file is full of errors. So I will close it as I guess this is the main problem for the seg fault.

![Screenshot (281)](https://user-images.githubusercontent.com/24510556/127488379-b53c3704-46fd-4f0b-96cf-002ee7bba6c8.png)
![Screenshot (280)](https://user-images.githubusercontent.com/24510556/127488429-a9056819-472f-4b05-8cab-7501f3e86f8b.png)

Try again with an error-free file and reopen it if necessary.

**Comment by picatchoof** [comment]
Hello All
indeed its a TS that I record using TSDuck From Eutelsat E7WA. (But I never Worked on  Central Europe)
What I mentioned from Central Europ is that the stream from the Satellite that cover Russia is broadcasting the same DVB MPE format. in this 36E it's about 50 Data Encapsulated Channel that Tricolor broadcast using this same DVB-MPE Protocol.
BTW the mpe2ts & dvb_ip_mpe2ts tools works really fine on the sample that I send in TSDuck issue. and works fine even for the TP 11158V5000 that I used before. on This TP the eti-tools works fine too.

BUT

My work today in this new service on The TP *11078MHz V SR=5000*.
I upload for you just a sample.
I found even using Windows tools it's outputs are corrupted.
I'm just looking to figure out that those issues are not due to my new Ubuntu Installation and tools.

Thanx a lot for you help.
